CharacterCreation/01_04_Species_Opaleites.txt

```diff
@@ -2,7 +2,21 @@
 Opaleites
 ==============================
 
-  Opaleites are an organic amphibious species that 
+    Opaleites are an organic amphibious species that evolved on a planet that 
+was mostly covered by marshes, seas and oceans. Their unique resperatory system
+lets them breathe under water as well as on land. An Opaleite's eyes are a 
+little better at seeing underwater then they are out of it.
+
+    Play as an Opaleite if you ...
+
+*  want to stick close to your team as if they were your schoolmates. 
```

Comment:
I'd move the "if you" down to this line. Makes for consistent parallel structure.